src/CommonITILTask.php

```diff
@@ -112,13 +112,18 @@ public static function canUpdate(): bool
         ));
     }
 
-
+    /**
+     * @return bool|int
```

Comment:
I guess `Session::haveRight()` should always return a bool. Maybe it could be fixed.

```suggestion
     * @return bool
```

Reply:
Indeed, it can be done in the main branch.